projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,223 @@
+              >
+                <div class="d-md-none cx-order-history-label">
+                  {{ 'orderHistory.startOn' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-order-history-value"
+                >
+                  {{ order?.firstDate | cxDate }}</a
+                >
+              </td>
+              <td
+                class="cx-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-order-history-label">
+                  {{ 'orderHistory.frequency' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-order-history-value"
+                >
+                  {{ order?.trigger.displayTimeTable }}
+                </a>
+              </td>
+              <td
+                class="cx-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-order-history-label">
+                  {{ 'orderHistory.nextOrderDate' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-order-history-value"
+                >
+                  {{ order?.trigger.activationTime | cxDate }}
+                </a>
+              </td>
+              <td
+                class="cx-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-order-history-label">
+                  {{ 'orderHistory.estTotal' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-order-history-value"
+                >
+                  {{ order?.subTotal.formattedValue }}</a
+                >
+              </td>
+              <td (click)="openDialog()">
+                <a (click)="openDialog()" class="cx-order-cancel btn btn-link">
+                  {{ 'orderHistory.cancel' | cxTranslate }}
+                </a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <!-- Select Form and Pagination Bottom -->
+        <div class="cx-order-history-sort bottom row">
+          <div
```

Comment:
remove this div as we don't want to have the sorting at the bottom of the history page anymore. We only want to have the sorting at the top

Reply:
Jeanne asked me the same, but this change will create an inconsistency with the order history page. She was ok with deferring this to later when everything can be addressed together.

Reply:
no it won't because I'm changing it in the original one as well, therefore I mentioned it and to remove it here as well 

https://github.com/SAP/spartacus/issues/8969